bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/link/DefaultPathProcessorTest.java

```diff
@@ -117,7 +117,7 @@ void testVanityConfig() {
         DefaultPathProcessor underTest = context.registerInjectActivateService(new DefaultPathProcessor(), ImmutableMap.of(
                 "vanityConfig", "shouldBeDefault"));
         assertEquals("/content/site1/en.html", underTest.map(page.getPath() + HTML_EXTENSION, context.request()));
-        assertEquals("https://example.org/content/links/site1/en.html", underTest.externalize(page.getPath() + HTML_EXTENSION, context.request()));
+        assertEquals("https://example.org/content/site1/en.html", underTest.externalize(page.getPath() + HTML_EXTENSION, context.request()));
```

Comment:
There's a missing `/links` which puzzles me, who is removing it and why? Is this a breaking change?

Reply:
This is caused by a change in the AEM Mocks (https://github.com/wcm-io/io.wcm.testing.aem-mock/issues/44), which has been added in AEM Mocks 5.5.4